src/app/factory-surface.module.css

```
@@ -0,0 +1,51 @@
+  --secondary: #161616;
+  --secondary-foreground: #f7f7f4;
+  --muted: #121212;
+  --muted-foreground: #a3a3a3;
+  --accent: #123e31;
+  --accent-foreground: #c8f5df;
+  --border: rgb(255 255 255 / 0.11);
+  --input: rgb(255 255 255 / 0.14);
+  --ring: #ff775f;
+  --factory-bg: #050505;
+  --factory-line: rgb(255 255 255 / 0.07);
+  background: var(--factory-bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)